jfrog-cli/artifactory/utils/buildinfoproperties.go

```diff
@@ -193,6 +207,25 @@ func CreateBuildInfoPropertiesFile(buildName, buildNumber string, config *viper.
 	return propertiesFile.Name(), nil
 }
 
+// If HTTP_PROXY env variable presented, adds to the config to deploy Gradle/Maven artifacts via proxy
```

Comment:
I suggest rephrasing the comment to:
// If the HTTP_PROXY environment variable is set, add to the config proxy details.

Reply:
Done